Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -1,97 +1,124 @@
+- I collaborate with managers and lead software engineers to identify and find ways to fill any gaps in technical skills required in my teams.
 
 <details>
 <summary>Examples</summary>
 
 - I was consistently in demand for design and code review.
-- I am actively mentoring multiple Octopus engineers.
+- I became aware that I was a bottleneck and worked proactively with managers to uplift and empower other people to make the team more effective.
+- I am actively mentoring multiple Octopus engineers, including a Lead Software Engineer aspiring to become a Principal Software Engineer.
 - Engineers at Octopus seek out to work on my teams to learn from me.
 - After reviewing a code contribution from an engineer I saw could be improved, I worked directly with that engineer and their manager or team leader to uplift their skills in the identified growth area.
 - I helped a team make a course correction based on new information or ideas.
 - I provided appropriate technical freedom, and framing, that allowed the Lead Software Engineers on my teams to thrive independently.
+- I have elevated/shifted my focus significantly because the team(s) I support have been uplifted and empowered to the point they can deliver their mission with me as a sounding board.
 
 </details>
 
 ## 🧭 Culture and Leadership
 
-- I create an environment where others feel safe, by modeling vulnerability, thinking in the open, being wrong, and accepting corrections.
-- I generate energy within the teams I work with, motivating them to achieve great outcomes.
+- I am a proponent of [operational excellence](https://en.wikipedia.org/wiki/Operational_excellence) and [learning from incidents](https://www.learningfromincidents.io/about).
+- I get things right a lot of the time.
+- I create an environment where others feel safe to take risks and make mistakes, by modeling vulnerability, thinking in the open, being wrong sometimes, and accepting corrections.
+- I generate energy in the teams I work with, motivating them to achieve great outcomes.
 - I facilitate effective and timely decision making, especially when the decision affects multiple teams.
 - I build solutions that improve the productivity and/or quality of work of many people/teams.
+- I recognise patterns of organisational dysfunction, by observing data points, amplifying these signals, and advocating effectively for improvement.
+- I am an advisor or contributor to the leadership teams in my group.
 - I set a positive example throughout the company for quality, accountability, and responsibility.
-- I actively share knowledge relevant to product delivery teams.
 - I help define hiring standards and practices.
 
 <details>
 <summary>Examples</summary>
 
-- I drove an entire multiteam program from inception through to shipping code, without regular technical oversight.
+- During a post-incident review, I recognised a valuable lesson and found a way to disseminate the lesson throughout R&D.
```

Comment:
```suggestion
- During an incident postmortem, I recognised a valuable lesson and found a way to disseminate the lesson throughout R&D.
```

[nitpick] Hoping we can settle on [consistent language](https://octopushq.atlassian.net/wiki/spaces/RND/pages/2414674935/How+to+run+a+valuable+Blameless+Postmortem).